src/install/oracle/whatsNext.mdx

```diff
@@ -0,0 +1,7615 @@
+---
+componentType: default
+headingText: Metrics collected by the integration
+---
+## Oracle DB instance settings [#instance-settings]
+
+The Oracle DB integration collects both Metrics(**M**) and Inventory(**I**) information. In the table, use the **Applies To** column for the settings available to each collection:
```

Comment:
Do horizontal tabs work w/i branching docs mdx files? One solution to shorten this area is to separate them somehow w/ horizontal tabs, tehn collapsers. It's pretty long